<?xml version="1.0"?>
<!-- SESSION ID: 1523559094 -->
<launch>

  <!-- Calibration parameters -->
  <rosparam command="load" file="$(find opt_calibration)/conf/camera_poses.yaml" />

  <!-- Network parameters -->
  <arg name="num_sensors"   default="2" />

  <arg name="sensor_0_id"   default="kinect_01" />
  <arg name="sensor_0_name" default="$(arg sensor_0_id)" />

  <arg name="sensor_1_id"   default="kinect_02" />
  <arg name="sensor_1_name" default="$(arg sensor_1_id)" />

  <!-- Launching calibration -->
  <node pkg="opt_calibration" type="opt_define_reference_frame" name="opt_define_reference_frame" output="screen">
    <rosparam command="load" file="$(find opt_calibration)/conf/camera_network.yaml" />

    <param name="num_sensors"           value="$(arg num_sensors)" />

    <param name="sensor_0/name"         value="/$(arg sensor_0_name)" />
    <param name="sensor_0/type"         value="pinhole_rgb" />
    <remap from="~sensor_0/image"       to="/$(arg sensor_0_name)/rgb_rect/image" />
    <remap from="~sensor_0/camera_info" to="/$(arg sensor_0_name)/rgb_rect/camera_info" />

    <param name="sensor_1/name"         value="/$(arg sensor_1_name)" />
    <param name="sensor_1/type"         value="pinhole_rgb" />
    <remap from="~sensor_1/image"       to="/$(arg sensor_1_name)/rgb_rect/image" />
    <remap from="~sensor_1/camera_info" to="/$(arg sensor_1_name)/rgb_rect/camera_info" />

  </node>

</launch>
